prac_10/testing.py

```diff
@@ -0,0 +1,65 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from prac_06.car import Car
+
+
+def repeat_string(s, n):
+    """Repeat string s, n times, with spaces in between."""
+    return " ".join([s] * n)
+
+
+def is_long_word(word, length=5):
+    """
+    Determine if the word is as long or longer than the length passed in
+    >>> is_long_word("not")
+    False
+    >>> is_long_word("supercalifrag")
+    True
+    >>> is_long_word("Python", 6)
+    True
+    """
+    return len(word) >= length
+
+
+def run_tests():
+    """Run the tests on the functions."""
+    assert repeat_string("Python", 1) == "Python"
+    # the test below should fail
+    assert repeat_string("hi", 2) == "hi hi"
+    test_car = Car()
+    assert test_car._odometer == 0, "Car does not set odometer correctly"
+    assert test_car.fuel == 0, "default fuel is not 0"
+    assert not test_car.fuel
+    test_car = Car(fuel=10)
+    assert test_car.fuel == 10, "fuel isn't added properly"
```

Comment:
Have you tried to test if the fuel has a new value? You need to assert and test what happens if you passed in a new value.